src/python/grpcio/grpc/beta/_server_adaptations.py

```diff
@@ -79,7 +79,8 @@ def protocol_context(self):
     return _ServerProtocolContext(self._servicer_context)
 
   def invocation_metadata(self):
-    return self._servicer_context.invocation_metadata()
+    return _common.cygrpc_metadata(
```

Comment:
Why's this being done? Is it to maintain compatibility with the current behavior of the Beta API?